controllers/topics-controller.js

```diff
@@ -0,0 +1,16 @@
+const selectTopics = require('../models/topics-model')
+
+
+function getTopics(req, res, next){
+
+    selectTopics()
+    .then((topics) => {
+        res.status(200).send(topics.rows)
+    })
+    .catch((err) => {
+        console.log(err)
```

Comment:
```
        console.log(err)
```
As a general rule try not to leave console.log's in as part of your PR's unless you want them there in production. 

If you do want to add some error logging at this point consider adding a 500 error handling middleware that can log them otherwise you'll have to do this every time.